packages/google_adsense/lib/src/adsense/ad_unit_widget.dart

```diff
@@ -58,13 +61,14 @@ class _AdUnitWidgetWebState extends State<AdUnitWidget>
   @override
   bool get wantKeepAlive => true;
 
-  static final web.ResizeObserver _adSenseResizeObserver = web.ResizeObserver(
+  web.ResizeObserver get _adSenseResizeObserver => web.ResizeObserver(
```

Comment:
This may now be creating multiple ResizeObservers on the DOM, whereas we only used to have a single one that was reused for all DOM elements.

I don't think this is a big deal performance-wise, but this could be refined a little bit so we keep the single static resize observer, but then expose an instance API to attach listeners to it like:

`void observe(Element, onResize)`, 

instead of directly using the `static _adSenseResizeObserver` directly.

This is not a blocking issue (but maybe a TODO for the future?)